build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/dsl/Publish.kt

```diff
@@ -34,6 +34,14 @@ private val ALLOWED_PUBLICATIONS = listOf(
     "android", // aws-crt-kotlin
     "codegen",
     "codegen-testutils",
+
+    // aws-sdk-kotlin:hll
+    "hll-codegen",
+    "dynamodb-mapper-codegen",
+    "dynamodb-mapper-schema-generator-plugin",
+    "dynamodb-mapper-schema-codegen",
+    "pluginMaven",
+    "dynamodb-mapper-schema-generatorPluginMarkerMaven",
```

Comment:
**Question**: What are these two?

Reply:
Actually, I was just able to remove both of these. Not sure when this changed, but at some point these were required.